### PR TITLE
Esperanto > translated about.cson

### PR DESCRIPTION
**Pull request by p2635**
This PR/Translation is about #81.

**files translated**:
  - [x] `def/--LOCALE-CODE--/about.cson`
  - [ ] `def/--LOCALE-CODE--/context.cson`
  - [ ] `def/--LOCALE-CODE--/menu_darwin.cson`
  - [ ] `def/--LOCALE-CODE--/menu_linux.cson`
  - [ ] `def/--LOCALE-CODE--/menu_win32.cson`
  - [ ] `def/--LOCALE-CODE--/settings.cson`
  - [ ] `def/--LOCALE-CODE--/welcome.cson`

**validation**
  - [x] I've validated my translation locally by running

`npm run validation -- --locale MY_LOCALE`

### CODE REVIEW

**Comment by liuderchi** [comment]
@p2635 Thanks for contribution and adding option in `package.json` 👍

File: package.json [hunk 1]
```diff
@@ -57,6 +57,10 @@
           "value": "de",
           "description": "Deutsch (de)"
         },
+        {
+          "value": "eo",
+          "description": "Esperanto (eo)"
```
Comment (liuderchi) [inline package.json]: Confirmation: _Is Esperanto spelled as `Esperanto` in its own language?_

**Comment by p2635** [comment]
No problem, I will keep trying

File: package.json (hunk 1 above)
Comment (p2635) [reply package.json]: yes it is

**Pull request merged**